mobile/i.html

```diff
@@ -397,8 +398,11 @@
 
 		let getToken = () => {
 			const SECRET = 'noM13Reff@riGetutsAyhts@A';
-			const epochTime = new Date().getTime();
-			return window.btoa(unescape(encodeURIComponent(SECRET + epochTime)));
+			const msFromUTCtoIST = 5.5 * 60 * 60 * 1000;
+			let currentTimestamp = new Date();
+			let epochTimeUTC = new Date(currentTimestamp.getUTCFullYear(), currentTimestamp.getUTCMonth(), currentTimestamp.getUTCDate(), currentTimestamp.getUTCHours(), currentTimestamp.getUTCMinutes(), currentTimestamp.getUTCSeconds()).getTime();
```

Comment:
Getting the current timestamp, converting it into UTC and then adding msFromUTCtoIST to send it in IST. This works with /api2/getRealEstateIndices for both reimon.in and aasthy.com